src/OmniSharp.Abstractions/Models/v1/MembersTree/FileMemberElement.cs

```diff
@@ -9,6 +9,10 @@ public class FileMemberElement : IComparable<FileMemberElement>
 
         public QuickFix Location { get; set; }
 
+        public int AttributeSpanStart { get; set; }
```

Comment:
Can we make these some kind of range or span type?

Reply:
We could pass both AttributeSpanStart and AttributeSpanEnd in a Span type class. But in that case we will also have to define this class in protocol.ts in omnisharp-vscode. Should that be done ?

Reply:
Protocol.ts defines a range type: https://github.com/OmniSharp/omnisharp-vscode/blob/master/src/omnisharp/protocol.ts#L450

Reply:
@rchande Yes it does, but the range has the start and end as `Point` type. And I couldn't spot a function in omnisharp-roslyn that converts from a `span` to `range`